torchaudio/transforms.py

```diff
@@ -1025,3 +1025,127 @@ def forward(self, waveform: Tensor) -> Tensor:
             hp_lifter_freq=self.hp_lifter_freq,
             lp_lifter_freq=self.lp_lifter_freq,
         )
+
+
+class Synth(torch.nn.Module):
+    r"""This effect can be used to generate fixed or swept frequency audio tones with various wave shapes,
+     or to generate noise.
+
+    Args:
+        sample_rate (int, optional): Sample rate of audio signal (Default: ``16000``).
+        duration (float, optional): Duration in seconds of audio to synthesise (Default: ``1.0``).
+        wave_type (str, optional): is one of sine, square, triangle, sawtooth, trapezium, exp, white noise
+         and brownian noise (Default: ``"sine"``).
+        freq (int or tuple, optional): are the frequencies at the beginning/end of synthesis in Hz (Default: ``440``).
+        chirp (str, optional): If `freq` defined the variation between to frequencies can be changed `linear`,
+            `square` or `exp` (Default: ```linear``).
+        amp (float, optional): Maximum amplitude (Default: ```1.0``).
+    """
```

Comment:
At one glance this class is not device/dtype-aware.
Could you rebase to the latest master and see if CI tests pass?
#616 added separate test for float32/float64.

Reply:
It passed without warning, so I will assume that is not necessary to do anything